pkg/piperutils/container.go

```diff
@@ -0,0 +1,41 @@
+package piperutils
```

Comment:
I would add this to the `docker` package instead of a generic `piperutils` one.

Reply:
good point, did so ...